examples/http-caching-client/http_caching_client.server.out

```diff
@@ -68,7 +68,7 @@ Host: localhost:9090
 User-Agent: curl/7.64.1
 Accept: */*
 
-# This time, the request is not served from the cache. The backend service is contacted. The `if-none-match` header sends the entity tag of the now stale response, so that the backend service may determine whether this response is still valid.
+# This time, the request is not served from the cache. The backend service is contacted. The if-none-match header sends the entity tag of the now stale response, so that the backend service may determine whether this response is still valid.
```

Comment:
```suggestion
# This time, the request is not served from the cache. The backend service is contacted. The `if-none-match` header sends the entity tag of the now stale response so that the backend service may determine whether this response is still valid.
```